src/Components/CurrentBenefits/CurrentBenefits.css

```diff
@@ -0,0 +1,44 @@
+.co-benefits-container {
+  margin: 1rem;
```

Comment:
Do we want to add a max width to this to be consistent with the rest of the app?